src/python/CRABClient/Commands/status.py

```diff
@@ -57,7 +60,7 @@ def __call__(self):
         if not rootDagId:
             failureMsg = "The task has not been submitted to the Grid scheduler yet. Not printing job information."
```

Comment:
This is shorthand for statusFailureMsg and it is a failure in the sense that the status cannot provide all of the information it normally would.